adv-rumors.lua

```diff
@@ -68,6 +61,17 @@ function renameChoice(text, choice)
     end
 end
 
+function getKeywords(choice)
+    local keywords = {}
+    for i, keyword in ipairs(choice.key_word) do
+        local keytext = dfhack.df2utf(keyword.value):lower()
```

Comment:
utf-8 is only for use when exporting text to the operating system (console or file). in-script text should not convert from the DF code page